newIDE/app/src/AssetStore/PrivateAssets/PrivateAssetPackInformationDialog.js

```diff
@@ -121,6 +139,20 @@ const PrivateAssetPackDialog = ({
                 label={<Trans>Cancel</Trans>}
                 onClick={onClose}
               />,
+              <RaisedButton
+                key="buy-asset-pack"
+                primary
+                label={
+                  !assetPack || isPurchaseDialogOpen ? (
```

Comment:
I feel like it would be safer user-emotion-wise to display `Loading` rather than `Processing` at dialog opening time

Reply:
yeah agree